src/MetricsUploader/MetricsUploaderWindowsTest.cpp

```diff
@@ -10,6 +10,12 @@
 #include "OrbitBase/Result.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/strings/ascii.h"
+#include "gmock/gmock-matchers.h"
+#include "gtest/internal/gtest-port.h"
```

Comment:
Nit: May be use <> style includes for everything external to Orbit.

Reply:
obsolete